core/vm/opcodes.go

```diff
@@ -200,11 +202,10 @@ const (
 	LOG4
 )
 
-// unofficial opcodes used for parsing.
```

Comment:
I removed those unofficial opcodes and they seem as leftovers. Let's verify they truly are

Reply:
These `unofficial opcodes used for parsing` were in the `0xb0` range. They have been replaced with `control flow ops`, namely `CALLF` and `RETF`, both ops via EIP-4750 (EOF - Functions). 

The original ops `PUSH`, `DUP`, and `SWAP` were shorthands for stack ops using scalar values (like `PUSH1`, `PUSH2`, etc.), and they are not used anywhere anymore.